test/e2e/page-objects/flows/login.flow.ts

```diff
@@ -42,6 +43,20 @@ export const loginWithBalanceValidation = async (
   await loginWithoutBalanceValidation(driver, password);
   const homePage = new HomePage(driver);
 
+  let skeleton: WebElement | undefined;
+  try {
```

Comment:
Some e2e tests load the home screen and then return to it. The first time a skeleton will appear but assuming that the set of enabled networks has not changed then the skeleton will not reappear. So we only need to assert that it disappears if it has first appeared.